api/swagger.yaml

```diff
@@ -3758,6 +3758,17 @@ definitions:
           or "Windows Server 2016 Datacenter"
         type: "string"
         example: "Alpine Linux v3.5"
+      OSVersion:
+        description: |
+          Version of the host's operating system
+
+          <p><br /></p>
+
+          > **Note**: The information returned in this field, including the
+          > its existence, and the formatting of values, should not be considered,
+          > stable, and may change without notice.
```

Comment:
"including the its existence" does not sound right to me

"considered, stable" -- looks like comma should not be here

Reply:
Good catch, fixed!